compiler/next/include/chpl/queries/stringify-functions.h

```diff
@@ -223,6 +234,51 @@ void operator()(std::ostream &stringOut,
 }
 };
 
+template<> struct stringify<unsigned long int> {
+void operator()(std::ostream &stringOut,
+                StringifyKind stringKind,
+                const unsigned long int val) const {
```

Comment:
I know we might move to an enable_if approach for these but if we keep them IMO it makes more sense to use `uint8_t` /  `int64_t` etc.

Reply:
Note that if we have specializations for int8_t / uint8_t we probably have to cast them to int to avoid printing as an ascii character